src/main/java/de/friday/sonarqube/gosu/plugin/rules/smells/UnnecessaryImportRule.java

```diff
@@ -52,8 +55,11 @@ public void exitPackageDeclaration(GosuParser.PackageDeclarationContext context)
     @Override
     public void exitUsesStatement(GosuParser.UsesStatementContext context) {
         if (isNamespaceAvailable(context)) {
-            final String usesStatement = context.namespace().getText();
-            checkUnnecessaryImport(usesStatement, context);
+            final UsesStatement usesStatement = new UsesStatement(
```

Comment:
Why can't you use `new UsesStatement(context)` ?
It could be really helpful in #87 impl.